common/src/main/java/com/wynntils/features/map/MainMapFeature.java

```diff
@@ -46,7 +46,7 @@ public class MainMapFeature extends Feature {
     public final Config<Float> playerPointerScale = new Config<>(1.5f);
 
     @Persisted
-    public final Config<Float> poiScale = new Config<>(1f);
```

Comment:
Do we need an upfixer?

Reply:
I'd like to think that an explicit "reset" won't hurt with the new changes. But probably I can add it too..

Reply:
Ok, fair enough, old values might not really apply anymore.